src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -494,15 +523,15 @@ def continuous_pq_update(self, table: pyarrow.Table) -> None:
                 ),
                 ignore_errors=True,
             )
-            self.error_files += self.archive_files
+            # self.error_files += self.archive_files
```

Comment:
why are we taking this out?

Reply:
Was a holdover from testing on dev, will revert